src/components/LoadingSpinner/styles.scss

```diff
@@ -0,0 +1,27 @@
+.container {
```

Comment:
Does scss change class names after the build? If not having generic names for the classes will create awkward problems when two or more same-named classes are used on the page.

Reply:
I thought that scss package handles class names like how module.css handles class names but idk if that will be the case with scss should I play safe and change class names ?

Reply:
It doesn't handle it
https://discuss.codecademy.com/t/if-we-have-multiple-css-files-with-the-same-class-name-which-ones-properties-is-applied/389104

Reply:
Make sure to give specific class names